web-api/migration-terraform/main/lambdas/migration-segments.js

```diff
@@ -3,6 +3,9 @@ const createApplicationContext = require('../../../src/applicationContext');
 const {
   migrateItems: migration0025,
 } = require('./migrations/0025-docket-entry-received-at-strict-timestamp');
+const {
+  migrateItems: migration0027,
```

Comment:
Named this 0027, assuming it might get merged alongside the other PR with 0026, but we can address this differently as needed if that's not the case.